app/views/people/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Here's a list of Task Owners!</h1>
+<table class="table table-striped table-hover table-users">
+
+
+
+<thead>
```

Comment:
This view looks like it does what you want it to do, but don't forget to get rid of empty lines (3 & 4) and then indent the table head and table body under <table>